packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -18,6 +18,11 @@ export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorT
    */
   rootVerticalPadding?: number;
 
+  /**
+   * Left padding of the root view
+   */
+  rootLeftPadding?: number;
+
```

Comment:
Should this be `rootStartPadding` for RTL support?

Reply:
Removed this token